### PR TITLE
Add QuizUp to adopters list

### PR DESCRIPTION
**Pull request by arnarthor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,4 +136,4 @@ limitations under the License.
 [finagle]: https://github.com/twitter/finagle
 [circe]: https://github.com/travisbrown/circe
 [jackson]: http://jackson.codehaus.org/
-[argonaut]: http://argonaut.io/
\ No newline at end of file
+[argonaut]: http://argonaut.io/
```
Comment (arnarthor) [inline README.md]: I have no idea why it says this line has changed ¯\_(ツ)_/¯

**Comment by vkostyukov** [comment]
Thanks @arnarthor!

A couple of years ago I was playing this awesome game called QuizUp on my phone and I didn't even imagine that some day you will be using an OSS project I work on!

PS: I'd also appriciate a PR [against Finagle](https://github.com/twitter/finagle/blob/master/ADOPTERS.md).

**Comment by arnarthor** [comment]
Will add one to Finagle as well!

**Comment by vkostyukov** [comment]
Thanks again @arnarthor! I'm merging this.

**Pull request merged**